### PR TITLE
Add `clearLaunchScreenCache()`

### PR DESCRIPTION
**Pull request by IgorRosocha**
#### Checklist

- [X] Updated CHANGELOG.md.
- [X] Added tests (if applicable)

### CODE REVIEW

File: ACKategories/UIApplicationExtensions.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//  Copyright © 2019 Ackee, s.r.o. All rights reserved.
+//
+
+import UIKit
+
+public extension UIApplication {
+    /// Completely clear launch screen cache
+    func clearLaunchScreenCache() {
+        do {
+            try FileManager.default.removeItem(atPath: NSHomeDirectory()+"/Library/SplashBoard")
```
Comment (LukasHromadnik) [inline ACKategories/UIApplicationExtensions.swift]: Please add spaces around the plus `+` sign 🙂

Comment (IgorRosocha) [reply ACKategories/UIApplicationExtensions.swift]: Definitely 🙂

File: ACKategories/UIApplicationExtensions.swift [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+//  ACKategories
+//
+//  Created by Igor Rosocha on 12/16/19.
+//  Copyright © 2019 Ackee, s.r.o. All rights reserved.
+//
+
+import UIKit
+
+public extension UIApplication {
+    /// Completely clear launch screen cache
```
Comment (janmisar) [inline ACKategories/UIApplicationExtensions.swift]: Please add a short story what is this method for and when and why should I use it and also the link to article 🙏

Comment (IgorRosocha) [reply ACKategories/UIApplicationExtensions.swift]: No problem 🙂

**Pull request merged**